bundles/AdminBundle/src/Controller/Admin/IndexController.php

```diff
@@ -250,6 +250,10 @@ protected function buildPimcoreSettings(Request $request, array &$templateParams
 
             // document types
             'document_types_configuration' => Document::getTypesConfiguration(),
+            'document_search_types' => Document::getTypes(),
+            'document_valid_types' => array_values(array_filter(Document::getTypes(), function ($type) {
+                return $type != 'folder';
```

Comment:
```suggestion
                return $type !== 'folder';
```
Let's make it as save as we can right? :)